.github/workflows/docker-build-and-push.yml

```diff
@@ -47,6 +47,16 @@ on:
         required: false
         default: 'https://dev.api.telescope.cdot.systems/v1/status'
         type: string
+      supabase_url:
+        description: 'The Supabase URL'
+        required: false
+        default: 'http://kong:8000'
+        type: string
+      anon_key:
+        description: 'Anon key used to initialize supabase client'
+        required: false
+        default: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJzZXJ2aWNlX3JvbGUiLAogICAgImlzcyI6ICJzdXBhYmFzZS1kZW1vIiwKICAgICJpYXQiOiAxNjQxNzY5MjAwLAogICAgImV4cCI6IDE3OTk1MzU2MDAKfQ.DaYlNEoUrrEn2Ig7tqibS-PHK5vgusbcbo7X36XVt4Q'
```

Comment:
It feels weird to have this key hard-coded.  I know it's not technically a secret, but this seems like a bad idea.

Can you create a new GitHub Actions secret in the Telescope repo named `SUPABASE_ANON_KEY` and paste this key in there.  We'll update it later on staging and production when we turn Supabase on for real.

Then we'll need to move `anon_key` down to the `secrets` section below.